test/integration/accounts.ts

```diff
@@ -31,7 +31,7 @@ import {
 } from '../../src/utils/errors'
 
 describe('Accounts', function () {
-  let aeSdk, aeSdkWithoutAccount
+  let aeSdk: any, aeSdkWithoutAccount: any
```

Comment:
@davidyuk Do you think we need a global type for the SDK? we may have better clarity once stampit is removed.
@Kalovelo